code/_helpers/logging.dm

```diff
@@ -80,10 +80,14 @@
 /proc/log_say(text)
 	if (config.log_say)
 		game_log("SAY", text)
+	if(usr)
+		usr.client.say_log += "([time_stamp()]) - [text]"
```

Comment:
This could also do with a coordinate and area stamp, same with emotes. Sorry for being vague in the original review, but it shouldn't be too much trouble, I hope.

Reply:
Hm, on one hand I see the point, on the other hand just how spammy do we want these logs to be? I will add location to say, but I'm on the edge with adding coords